docs/experience/longtasks.md

```diff
@@ -0,0 +1,63 @@
+title: Monitoring Browser Long Tasks
+description: Gathering information about long tasks during hard and soft page loads
+
+Experience collects information about long tasks that happen during page loads. According to the [RAIL](https://developers.google.com/web/fundamentals/performance/rail) Model, tasks that are running for longer than **50ms** and blocking the main thread are considered long and are recorded by Sematext Cloud. 
+
+Long tasks can cause issues such as:
+
+* Clunky animations and scrolling affecting user experience
+* High input latency
+* Long delay in events handling
+* Delayed time when your application becomes interactive
+
+Thus, it is critical to monitor long tasks executed as part of your application's handling of user requests.
+
+## Page Load Overview
+The aggregated data about long tasks are visible on the page loads overview of your Experience App. You can check the number of long tasks and their average duration:
+
+<img
+  class="content-modal-image"
+  alt="Page Loads Overview"
+  src="../../images/experience/longtasks/overview.png"
+  title="Page Loads Overview"
+  width=778
+/>
+
+
+## Page Load Details
+In addition to long tasks charts on the page loads overview, each individual page load details screen also contains information about long tasks detected during both hard and soft page loads. Each long task is described by its **name**, **timestamp**, and **duration**.
+
+The browser doesn't point us to a particular code in your application, but the **name** of the long task can point to an event of the page load that was executed for a longer period of time. The **name** value can take one of the following values:
```

Comment:
The browser doesn't point us to a particular code in your application, but the **name** of the long task can point to an event of the page load that was executed for a longer period of time.
=>
The browser doesn't point us to a particular piece of code in your application, but the **name** of the long task can point to an event of the page load that was executed for a longer period of time.

Reply:
Adjusted, thanks!